### PR TITLE
Animated pulse laser loader

### PR DESCRIPTION
**Pull request by ubertpendragon**
Adds a small effect to the pulse laser loader when fired

### CODE REVIEW

File: pulselaser.xml [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    <BrokenSprite texture="Content/Items/Weapons/Loaders.png"  sourcerect="163,563,153,351" origin="0.5,0.5" offset="5,6" depth="0.77" fadein="true" maxcondition="99"/>
+    <UpgradePreviewSprite texture="Content/UI/WeaponUI.png" sourcerect="208,968,32,46" origin="0.5,0.5" />
+    <DecorativeSprite name="Pulse Laser Loader Frame Back" texture="Content/Items/Weapons/Loaders.png" depth="0.8" sourcerect="31,569,117,200" origin="0.5,0.5" offset="0,-68" />
+    <LightComponent range="10.0" lightcolor="255,255,255,0" alphablend="true" pulsefrequency="1" pulseamount="0.2" powerconsumption="0" IsOn="false" castshadows="false" allowingameediting="false">
+      <sprite texture="Content/Items/Weapons/Loaders.png" sourcerect="0,791,165,358" depth="0.1" origin="0.5,0.5" alpha="1.0" />
+      <StatusEffect type="OnUse" target="This,Contained" IsOn="True">
+        <ParticleEmitter particle="pulselasermist" particleamount="2" anglemin="70" anglemax="110" scalemin="0.5" scalemax="0.5" distancemin="0" distancemax="20" velocitymin="50" velocitymax="100" drawontop="true"/>
+		<Conditional condition="gt 0.0" TargetContainedItem="true" />
```
Comment (Regalis11) [inline pulselaser.xml:69]: Messed up indentation

**Review by Regalis11 — CHANGES_REQUESTED** [review]
There's some indentation issues in the XML

File: pulselaser.xml (hunk 1 above)
Comment (ubertpendragon) [reply pulselaser.xml]: Fixed

**Comment by Regalis11** [comment]
Merged to our private repo now (with some minor adjustments)

**Pull request closed without merging**